students/ThanhNhan/lesson03/slicing_lab.py

```diff
@@ -0,0 +1,66 @@
+#!/usr/bin/env python
+
+def exchange_first_last (seq):
```

Comment:
Go ahead and remove the space between the function name and the open parentheses.  Same for the print statement below: print(seq)